### PR TITLE
#102: add customers sync method

### PR DESCRIPTION
**Pull request by vladarefiev**
PR for https://github.com/timelesslounge/timelessis/issues/102
- Add fetch Customer method to poster
- Leave puzzle for celery (async/cron jobs) set up

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#193](https://github.com/timelesslounge/timelessis/issues/193) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#193](https://github.com/timelesslounge/timelessis/issues/193) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/4d0766ed-20b7-46f2-92be-73652e6bc43c); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by vladarefiev** [comment]
@amihaiemil Take a look pls when you will be able for

File: tests/test_poster.py [hunk 1]
```diff
@@ -14,4 +14,7 @@ def test_locations(self):
         assert (self.poster.locations()["data"] == "testData")
 
     def test_tables(self):
-        assert (self.poster.tables()["data"] == "testTables")
\ No newline at end of file
+        assert (self.poster.tables()["data"] == "testTables")
+
+    def test_customers(self):
```
Comment (amihaiemil) [inline tests/test_poster.py]: @vladarefiev shouldn't you instruct the mock Poster server to return "testCustomers" for "customers" call? I see it's made so for test_locations and test_tables

File: tests/poster_mock.py [hunk 2]
```diff
@@ -37,23 +38,28 @@ class PosterServerMock(BaseHTTPRequestHandler):
...
                 self.send("locations")
                 return
+
             if re.search(self.TABLES_PATTERN, self.path):
                 self.send("tables")
                 return
 
+            if re.search(self.CUSTOMERS_PATTERN, self.path):
+                self.send("customers")
+                return
```
Comment (amihaiemil) [inline tests/poster_mock.py]: @vladarefiev Are there any unit tests for ``poster_mock.py`` itself? If so, can you add a case for the customers?

File: tests/test_poster.py (hunk 1 above)
Comment (vladarefiev) [reply tests/test_poster.py]: @amihaiemil good catch! thanks

File: tests/poster_mock.py (hunk 2 above)
Comment (vladarefiev) [reply tests/poster_mock.py]: @amihaiemil No, there are not. `poster_mock` is imported in `test_poster` and there partially we test it.

**Comment by vladarefiev** [comment]
@amihaiemil fixed, take a look pls

**Comment by amihaiemil** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job [#193](https://github.com/timelesslounge/timelessis/issues/193) is not in the agenda of @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), can't inspect

**Comment by 0crat** [comment]
The job [#193](https://github.com/timelesslounge/timelessis/issues/193) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by amihaiemil** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/timelesslounge/timelessis/issues/193#issuecomment-462730407))

@amihaiemil This is what I know about this job in [CF9LG1YCA](https://www.0crat.com/p/CF9LG1YCA), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#193](https://github.com/timelesslounge/timelessis/issues/193) is not in scope
  * The job is waiting quality review verdict by @ypshenychka[/z](https://www.0crat.com/u/ypshenychka) for 1hr
  * @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) will receive 15 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/CF9LG1YCA?q=%7Bjob%3A%27gh%3Atimelesslounge%2Ftimelessis%23193%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by amihaiemil** [comment]
@ypshenychka This PR waits for your review, apparently (see above comment). I think there was a problem and 0crat didn't mention you (see [here](https://github.com/timelesslounge/timelessis/pull/193#issuecomment-462711975))

**Comment by ypshenychka** [comment]
@amihaiemil According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by amihaiemil** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@amihaiemil thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**